doc/manuals/user/metadata.md

```diff
@@ -76,52 +76,6 @@ Note that, from the point of view of [subscriptions](walkthrough_apiv2.md#subscr
 changing the metadata of a given attribute is considered a change even
 though the attribute value itself hasn't changed.
 
-## Updating metadata
-
-When an attribute is updated the following rules apply:
-
-* Metadata included in the attribute update request *not previously existing* are added
-  to the attribute
-* Metadata included in the attribute update request *previous existing* are updated
-  in the attribute
-* Existing metadata not included in the request are not touched in the attribute (i.e. they keep the
-  previous value).
-
-For instance, let's consider an attribute `temperature` with metadata `unit` and `avg` which values
-are at the present moment:
-
-* `unit`: `"celsius"`
-* `avg`: `25.4`
-
-and Context Broker receives a request like this:
-
-```
-PUT /v2/entities/E/attrs/temperature
-{
-  "value": 26,
-  "type": "Number",
-  "metadata": {
-    "avg": {
-      "value": 25.6,
-      "type": "Number"
-    },
-    "accuracy": {
-      "value": 98.7,
-      "type": "Number"
-    }
-  }
-}
-```
-
-After processing the update, the metadata at the attribute `temperature` would be:
-
-* `unit`: `"celsius"` (existing and not touched by the request)
-* `avg`: `25.6` (existing but touched by the request)
-* `accuracy`: `98.7` (metadata added by the request)
-
-The rationale behind the "stikyness" of metadata in this default behaviour is described in
-more detail in [this issue at Orion repository](https://github.com/telefonicaid/fiware-orion/issues/4033)
-
 ### `overrideMetadata` option
```

Comment:
Should this part be included into API "Metadata update semantics"?

Reply:
Yes, I think so. The `overrideMetadata` flag modify the way in which metadata update semantics works, so this part is meaningful within"Metadata upddate semantics" section (maybe as a sub-section)

Reply:
Done in 976e5095915db8cb13a51ccf8ea74aa9b2726da3

Reply:
Embedded the rest of the document here:
a0828ff5bcc5835ada73006e51a551908b7a2d26